src/Agent.Worker/Handlers/AzurePowerShellHandler.cs

```diff
@@ -252,6 +252,12 @@ private void OnDataReceived(object sender, ProcessDataReceivedEventArgs e)
                                                                                       new Tuple<string, string>("Scope", "Global"),
                                                                                   }));
 
+			Trace.Verbose("AzurePowerShellHandler.UpdatePowerShellEnvironment - AddCommand(Set-UserAgent)");
```

Comment:
Can you change this to the compiled-syntax-equivalent of the following script syntax:
```powershell
if ((Get-Command Set-UserAgent)) { Set-UserAgent }
```

The psm1 file doesn't live in this repo. It gets pulled in as an external dependency. Unless you're change is going in with the commit moving the externals forward, it needs to handle both ways.